src/main/resources/swagger/api-docs.yaml

```diff
@@ -2859,3 +2882,52 @@ definitions:
       billingAccount:
         type: string
         description: the id of the billing account to use, must start with 'billingAccounts/'
+
+  ErrorReport:
+    description: ''
+    required:
+      - source
+      - message
+      - causes
+      - stackTrace
+    properties:
+      source:
+        type: string
+        description: service causing error
+      message:
+        type: string
+        description: what went wrong
+      statusCode:
```

Comment:
Hm - good question. I copy/pasted this directly from rawls; since this endpoint is a passthrough, it's going to return whatever rawls does.

In Rawls, ErrorReport.statusCode is an `Option[StatusCode]` ... maybe because we can cause an error internally (e.g. database connection failure) without an HTTP status code?